keanacobarde/INDIVIDUAL-ASSESSMENT-vocab-YOU-lary
# Kc stylingstretchprt2 stretch

## Description
- Revamped styling for login page and used Imgur to display crafted gif
- Fixed card spacing 
- Completed Readme
- Created Loom walkthrough

## Related Issue
#16 

## Motivation and Context
- The Readme is essential for a portoflio
- Styling changes help make login page more aesthetically pleasing
- Changing card spacing makes cards more aesthetically pleasing

## How Can This Be Tested?
Pull PR branch to local machine, run npm start, and verify that the login page looks the way as it needs to, cards are spaced evenly, and look at GitHub to make sure readme is done 

## Screenshots (if appropriate):

## Types of changes
